### PR TITLE
Feature/g3 cejzax personal ids

### PR DESCRIPTION
**Pull request by leomendoza123**

### CODE REVIEW

**Comment by leomendoza123** [comment]
[[speedup] Bio: Person identifiers modal (3)](https://trello.com/c/Q8Z83GnY/7215-speedup-bio-person-identifiers-modal-3)

File: src/app/cdk/side-bar/modals/modal-person-identifiers/modal-person-identifiers.component.html [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+          >
+            <mat-icon class="material-icons-outlined large-material-icon"
+              >delete</mat-icon
+            >
+          </button>
+        </div>
+      </div>
+    </div>
+
+    <div class="row"></div>
```
Comment (DanielPalafox) [inline src/app/cdk/side-bar/modals/modal-person-identifiers/modal-person-identifiers.component.html:106]: I don't think this is necessary

File: src/app/cdk/side-bar/modals/modal-person-identifiers/modal-person-identifiers.component.scss-theme.scss [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+@import 'src/assets/scss/material.orcid-theme.scss';
+
+@mixin modal-country-theme($theme) {
+  $primary: map-get($theme, primary);
+  $accent: map-get($theme, accent);
+  $warn: map-get($theme, accent);
+  $foreground: map-get($theme, foreground);
+  $background: map-get($theme, background);
+
+  ::ng-deep .country-input-line {
```
Comment (DanielPalafox) [inline src/app/cdk/side-bar/modals/modal-person-identifiers/modal-person-identifiers.component.scss-theme.scss:11]: Can you rename this or remove it, if it is not used

File: src/app/cdk/side-bar/modals/modal-person-identifiers/modal-person-identifiers.component.ts [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+    public dialogRef: MatDialogRef<ModalComponent>,
+    private _recordPersonalIdentifiers: RecordPersonIdentifierService,
+    private _changeDetectorRef: ChangeDetectorRef,
+    private _platform: PlatformInfoService
+  ) {}
+
+  addedEmailsCount = 0
+  personIdentifiersForm: FormGroup = new FormGroup({})
+  personIdentifiers: Assertion[]
+  personIdentifiersMap: { [key: string]: Assertion }
```
Comment (DanielPalafox) [inline src/app/cdk/side-bar/modals/modal-person-identifiers/modal-person-identifiers.component.ts:40]: Could you remove this unused variable

File: src/locale/properties/side-bar/side-bar.en.properties [hunk 4]
```diff
@@ -25,3 +25,7 @@ side-bar.weHaveSentAEmail=We’ve sent a verification email to
 side-bar.pleaseCheckYour=Please check your email inbox (or maybe your spam folder) for an email with a link to complete the process.
 side-bar.anotherEmail=Add another email address
 side-bar.perOrcid=Per the ORCID
+side-bar.personalIdentifiers=Personal identifiers
+side-bar.myPersonIdentifiers=My person identifiers
+side-bar.person-identifiers-description=Person identifiers are other identifiers that systems use to identify you, such as ResearcherID or Scopus Author ID. These identifiers can only be added by trusted organizations linked with your ORCID record. Find out more on our
```
Comment (DanielPalafox) [inline src/locale/properties/side-bar/side-bar.en.properties:30]: Could you use camel case like in the other properties

**Review by DanielPalafox — CHANGES_REQUESTED** [review]

**Review by DanielPalafox — APPROVED** [review]

**Pull request merged**